server/lib/nl/detection/place.py

```diff
@@ -95,14 +98,19 @@ def detect_from_query_dc(orig_query: str,
 
   # Set PlaceDetection.
   places_str = []
+  entities_str = []
   nonplace_query_parts = []
   for p in query_parts:
+    if p.dcid in resolved_entity_dcids:
+      # The query part is an entity dcid so add its original string to entities_str
+      entities_str.append(p.substr)
     if p.dcid in resolved_place_dcids:
```

Comment:
hm, it shouldn't happen where a place dcid is in both resolved_entity_dcids AND resolved_place_dcids, but technically if it does happen, shouldn't we include that string as both a string for place and a string for entity? since we will be including that dcid for both places found and entities found? wdyt

Reply:
ok, including in both sg.